src/compiler/shame.extern.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @type {{buildDom: BuildDomDef}}
+ */
+var buildDom$$module$src$builtins$amp_layout$amp_layout;
+
+/**
+ * @type {{buildDom: BuildDomDef}}
+ */
+var buildDom$$module$extensions$amp_fit_text$0_1$amp_fit_text;
```

Comment:
```suggestion
/** @type {BuildDomDef} */
var buildDom$$module$src$builtins$amp_layout$amp_layout;
/** @type {BuildDomDef} */
var buildDom$$module$extensions$amp_fit_text$0_1$amp_fit_text;
```

Reply:
I don't believe thats correct. The module has a named export for `buildDom`

Reply:
If you were defining the type for `default$$module$extensions$amp_fit_text$0_1$amp_fit_text`, this would be correct. For the named export `buildDom$$module$extensions$amp_fit_text$0_1$amp_fit_text` you're getting a `!BuildDomDef`

Reply:
Does this pass when you run `amp check-types target=src-compiler`?

Reply:
It does..but I agree with you and am confused why it is passing

Reply:
I don't think closure is handling the import type definitions correctly :/